components/kyma-environment-broker/cmd/cli/command/options.go

```diff
@@ -23,10 +23,17 @@ const (
 )
 
 const (
-	targetAccount    = "account"
-	targetSubaccount = "subaccount"
-	targetRuntimeID  = "runtime-id"
-	targetRegion     = "region"
+	accountTarget    = "account"
+	subaccountTarget = "subaccount"
+	runtimeIDTarget  = "runtime-id"
+	regionTarget     = "region"
+	planTarget       = "plan"
+)
+
+const (
+	azurePlan     = "azure"
+	azureLitePlan = "azure_lite"
+	trialPlan     = "trial"
```

Comment:
don't we want to support all the plans available under `internal/broker/plans.go`?